input/pagecontent/authentication.md

```diff
@@ -0,0 +1,169 @@
+### Introduction
+
+Authentication verifies the identity of entities, such as healthcare professionals and organizations, within healthcare systems.
+Verified identities support downstream processes such as secure authorization (access control) and accounting (audit logging).
+
+### Problem overview
+
+Healthcare professionals often require access to patient data managed by other organizations (e.g. hospitals). Legal and regulatory frameworks restrict data sharing, requiring strong verification of:
```

Comment:
The leans toward "getting data"-style use cases, but maybe rephrase/add "workflow"-style use cases (e.g. eOverdracht, ordering home monitoring, etc).